content/en/platform/corda/4.8/enterprise/node-explorer.md

```diff
@@ -132,7 +132,7 @@ To see a geographical view of your network, click **Network** in the left-hand s
 
 {{< note >}}
 
```

Comment:
In this file there's a link with _CorDapp's_ in it, so the apostrophe is messing with the formatting.